iodata/test/data/water_unrecognized_tag.wfx

```diff
@@ -0,0 +1,156 @@
+<Haha>
+This is a unrecognizable tag
+</Haha>
```

Comment:
I don't think we need a full wfx file to test an unexpected tag. It's better to add a smaller test file.